bin/monitor

```diff
@@ -3,7 +3,7 @@
 
 require_relative "../loader"
 
-MONITORABLE_RESOURCE_TYPES = [PostgresServer, MinioServer, GithubRunner, VmHost, Vm.where(~Sshable.where(id: Sequel[:vm][:id]).exists)]
+MONITORABLE_RESOURCE_TYPES = [VmHost, PostgresServer, Vm.where(~Sshable.where(id: Sequel[:vm][:id]).exists), MinioServer, GithubRunner]
```

Comment:
I think MinIO server is less ephemeral than a VM
```suggestion
MONITORABLE_RESOURCE_TYPES = [VmHost, MinioServer, PostgresServer, Vm.where(~Sshable.where(id: Sequel[:vm][:id]).exists), GithubRunner]
```

Reply:
And it's also a prerequisite for PG :D So IMHO, VmHost > MinIO > PG > Vm > Runner

Reply:
I put the MinIo later in the list because if it is down, it doesn't immediately create a customer impact.

Backup/Restore and new host initialization would not work, but user workloads are still running.